apps/src/templates/instructions/teacherFeedback/TeacherFeedback.jsx

```diff
@@ -280,7 +280,7 @@ export class TeacherFeedback extends Component {
             {viewAs === ViewType.Student &&
               this.renderCommentAreaHeaderForStudent()}
             {displayComment && (
-              <CommentArea
+              <Comment
```

Comment:
(unrelated) Is the `studentHasFeedback` property being used?

Reply:
Nope! Good catch